dts/arm/st/n6/stm32n6.dtsi

```diff
@@ -640,6 +642,19 @@
 			dma-offset = <0>;
 			status = "disabled";
 		};
+
+		xspi1: xspi@58025000 {
+			compatible = "st,stm32-xspi";
+			reg = <0x58025000 0x1000>;
+			interrupts = <170 0>;
+			clock-names = "xspix", "xspi-ker", "xspi-mgr";
```

Comment:
Nit: Make this prop required to be sure we error out early on misconfigurations.